docs/AIE Routing.md.md

```diff
@@ -0,0 +1,76 @@
+﻿# AIE Flows and Routing
```

Comment:
The name of this file is weird. (.md.md?)
In general, I'd avoid spaces in file names, if possible.

Reply:
I updated these docs with the changes you requested - the only thing I did not add was the dynamic ddr example since we don't have an example of that.